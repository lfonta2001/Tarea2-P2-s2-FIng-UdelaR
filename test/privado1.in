crearFecha 1/1/2021
crearProducto 70 productoprivado1 20
idProducto
imprimirProducto
liberarProducto
Fin
