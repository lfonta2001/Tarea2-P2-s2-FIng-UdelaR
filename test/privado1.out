Bienvenido al programa principal de Mercado FINGer. Por favor ingrese su comando:
1> Fecha creada en forma exitosa.
2> Producto creado en forma exitosa.
3> ID: 70
4> Producto: 70
productoprivado1
Precio: 20
Ingresado el: 1/1/2021
5> Producto liberado en forma exitosa.
6> Fin.
